### PR TITLE
go-jsonnet 0.15.0 (new formula)

### PR DESCRIPTION
**Pull request by FraBle**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/go-jsonnet.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  url "https://github.com/google/go-jsonnet/archive/v0.15.0.tar.gz"
+  sha256 "cddc3462151de4c5c65183b613895575db06297edd12a63e740f09e5e0ed98dd"
+  head "https://github.com/google/go-jsonnet.git"
+
+  depends_on "go" => :build
+
+  conflicts_with "jsonnet", :because => "both install binaries with the same name"
+
+  def install
+    ENV["GOPATH"] = buildpath
```
Comment (SMillerDev) [inline Formula/go-jsonnet.rb:13]: Is there a way to build without GOPATH?

File: Formula/go-jsonnet.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    bin_path = buildpath/"src/github.com/google/go-jsonnet"
+    contents = Dir["{*,.git,.gitignore}"]
+    bin_path.install contents
+    cd bin_path do
+      system "go", "build", "-o", bin/"jsonnet", "./cmd/jsonnet"
+      # jsonnetfmt will only be added with the next release
+      # system "go", "build", "-o", bin/"jsonnetfmt", "./cmd/jsonnetfmt"
+    end
```
Comment (chenrui333) [inline Formula/go-jsonnet.rb:22]: maybe this should be sufficient?
```suggestion
      system "go", "build", "-o", bin/"jsonnet", "./cmd/jsonnet"
      # jsonnetfmt will only be added with the next release
      # system "go", "build", "-o", bin/"jsonnetfmt", "./cmd/jsonnetfmt"
```

Comment (FraBle) [reply Formula/go-jsonnet.rb]: @chenrui333 thank you for your review and feedback, I've just updated PR.

File: Formula/go-jsonnet.rb (hunk 1 above)
Comment (FraBle) [reply Formula/go-jsonnet.rb]: @SMillerDev thank you for your review and feedback, I've just updated PR.

File: Formula/go-jsonnet.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/go-jsonnet.rb]: Looks good, thanks for the update!!

**Comment by chenrui333** [comment]
Thanks @FraBle for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**